x-pack/plugins/security_solution/server/endpoint/services/actions/action_details_by_id.ts

```diff
@@ -106,6 +109,13 @@ export const getActionDetailsById = async (
     throw new NotFoundError(`Action with id '${actionId}' not found.`);
   }
 
+  // get host metadata info with queried agents
+  const agentsInfo = await getAgentsMetadataInfo({
```

Comment:
so we are always going to include the `hosts` in the output, right? I though at one point we were discussing putting this behing a `expand` url param on the API

Reply:
Yeah, that makes sense. No need to pull the hostnames in the endpoint details and responder log. Will change

Reply:
https://github.com/elastic/kibana/pull/139379/commits/9d6759adac93091e16b2db888f7b29c3ab82dc2f#diff-fe47faa1b27954f966754419b04a6b6ea207203fe540862dad644197b9f3de4aL166-R206